crates/swc_ecma_minifier/tests/terser/compress/issue_597/NaN_and_Infinity_should_not_be_replaced_when_they_are_redefined/output.mangleOnly.js

```diff
@@ -1,3 +1,3 @@
-var t, i;
-Infinity.toString();
-NaN.toString();
+var t, r;
```

Comment:
Anyway, We should not rename it.

Reply:
Why not? All terser test cases are run in esm, it's fine to rename anything not exported.

Reply:
For
```js
var NaN = "zabcded"

console.log(NaN, NaN)
```
terser in `module: false` and `mangle: { toplevel: true }` would generate
```js
var o="zabcded";console.log(o,o);
```
I believe this is the right solution.

Reply:
Is this output expected? I expected this to be preserved in non-ESM mode, but maybe I was wrong

Reply:
And for
```js
var NaN = 1
console.log(NaN)
```
terser in `module: false` and `compress: { toplevel: true }` would generate
```js
console.log(1)
```

Reply:
> Is this output expected? I expected this to be preserved in non-ESM mode, but maybe I was wrong

No, this doesn't seem to be right.